components/vf-inlay/vf-inlay.scss

```diff
@@ -11,100 +11,53 @@
 
 @import 'vf-inlay.variables.scss';
 
-.vf-inlay {
-  display: grid;
-  grid-column: 1 / -1;
-  /* stylelint-disable  declaration-colon-space-after, indentation */
-  grid-template-columns:
-    minmax(var(--page-grid-gap), auto)
-    [main-start]
-      minmax(auto, 76.5em)
-    [main-end]
-    minmax(var(--page-grid-gap), auto)
-  ;
-  /* stylelint-enable */
-
-  &:last-of-type {
-    @include margin--block(bottom, $vf-inlay__spacing--bottom--narrow);
-
-    @media (min-width: 1024px) {
-      @include margin--block(bottom, $vf-inlay__spacing--bottom);
-    }
-  }
-
-  // overides vf-grid's natural inclination to be full width
-  & .vf-grid {
-    @media (max-width: 1299px) {
-      grid-column: main;
-    }
-  }
-
-  @media (min-width: 480px) {
-    // grid-column: 2 / -2;
-  }
-
-  @media (min-width: 1024px) {
-    --page-grid-gap: 36px;
-  }
-}
-
 .vf-inlay__content {
-  display: grid;
-  grid-column: 2 / 3;
-  /* stylelint-disable declaration-colon-space-after, indentation */
-  grid-template-columns:
-    minmax(var(--page-grid-gap), 1em)
-    [main-start]
-      minmax(auto, 100%)
-      minmax(auto, 22.125em)
-    [main-end]
-    minmax(var(--page-grid-gap), 1em)
-  ;
-  padding-bottom: calc(var(--page-grid-gap) / 2);
-  padding-top: 36px;
+  grid-column: 1 / -1;
 
-  @media (max-width: 1023px) {
+  .vf-inlay > & {
+    display: grid;
+    grid-gap: var(--page-grid-gap);
+    /*stylelint-disable*/
     grid-template-columns:
-      var(--page-grid-gap)
-      [main-start]
-        minmax(auto, 100%)
-        minmax(auto, 22.125em)
-      [main-end]
-      var(--page-grid-gap)
+      [inlay--full-start]
+        1fr
+      [inlay--full-end]
     ;
-  }
-  /* stylelint-enable */
 
+    @media (min-width: $vf-breakpoint--lg) {
+      grid-template-columns:
+        [inlay--full-start inlay--main-start]
+          1fr
+        [inlay--main-end inlay--sidebar-start]
+          minmax(auto, var(--embl-grid-sidebar-width, 21.25rem)
```

Comment:
I've added a `--embl-grid-sidebar-width` custom property here. 

So when I give `embl-grid` layout the extra 80px we have now that `vf-body` is 1300px width. We can give this column a bit more space too

Reply:
There's a yearning here to not have "embl" in the component, but probably not important as this component will slowly fade away.

Reply:
hmm, I didn't think of this 'cross signals'.

The only time the sidebar seems to be used is for the 'intro' or in place of the 'intro'

Perhaps we leave this for now and then I can come back to it.

Ideally - things like that should be created using `vf-grid` and the relevant span classes - to avoid creating a component just to affect the layout.

Reply:
leaving this in place for now. The CSS custom property comes as part of `vf-sass-config` so it shouldn't get missed.